ethereum/jsonrpc/src/main/java/tech/pegasys/pantheon/ethereum/jsonrpc/internal/methods/DebugAccountRange.java

```diff
@@ -102,7 +102,13 @@ public JsonRpcResponse response(final JsonRpcRequest request) {
                   .collect(
                       Collectors.toMap(
                           account -> account.getAddressHash().toString(),
-                          account -> account.getAddress().toString())),
+                          account -> {
```

Comment:
(optional) You can make this more concise with something like:
```suggestion
                          account -> account.getAddress().map(Address::toString()).orElse(null)
```